deploy/helm/elastic-agent/values.yaml

```diff
@@ -287,55 +287,74 @@ kubernetes:
       # -- system metric stream vars
       # @section -- 2 - Kubernetes integration
       vars: {}
+system:
+  # -- enable System integration.
+  # @section -- 4 - System integration
+  enabled: false
```

Comment:
I do feel like default here is a better choice.

Reply:
Resolving in favour of https://github.com/elastic/elastic-agent/pull/5855#discussion_r1816742053 🙂